src/components/interaction-designer/resource-editors/ResourceVariantTextEditor.vue

```diff
@@ -3,13 +3,21 @@
     <div
       class="content-editor"
       :class="{'content-editor-selected': !!content}">
-      <textarea
-        v-model="content"
-        v-focus="isSelected"
-        :placeholder="`flow-builder.enter-${resourceVariant.content_type.toString().toLowerCase()}-content` | trans"
-        class="form-control"
-        @focus="select"
-        @blur="deselect" />
+
+      <div class="input-group">
+        <div v-if="label" class="input-group-prepend">
+          <span class="input-group-text">{{label}}</span>
+        </div>
+
+        <textarea
+          v-model="content"
+          v-focus="isSelected"
+          :placeholder="placeholder || `flow-builder.enter-${resourceVariant.content_type.toString().toLowerCase()}-content` | trans"
+          class="form-control"
+          rows="1"
```

Comment:
Defaulted to single row `textarea` which appears as an expandable input[type=text]. This allows us to reuse this component for each choice on the sidebar as a single line input.